darwin/cli.py

```diff
@@ -63,7 +64,9 @@ def _run(args: Namespace, parser: ArgumentParser) -> None:
 
     # Authenticate user
     if args.command == "authenticate":
-        api_key = os.getenv("DARWIN_API_KEY")
+        api_key = os.getenv("DARWIN_API_KEY") or args.api_key
+        default_team = os.getenv("DARWIN_TEAM") or args.default_team
+        datasets_dir = os.getenv("DARWIN_DATASETS_DIR") or args.datasets_dir
         if api_key:
             print("Using API key from DARWIN_API_KEY")
```

Comment:
When I pass an API key with `--api_key`, I see `Using API key from DARWIN_API_KEY`

Is this technically incorrect, since it's actually from a CLI arg?

Reply:
Same goes for the `--default_team` and `--dataset_dir` flags

Reply:
updated for that